File: src/main/resources/js/testFeed.js
var userList = [
    {id: 1, name: "Nimal"},
    {id: 2, name: "Kamal"},
    {id: 2, name: "Upul"}
    ];
vertx.setPeriodic(2000, function(id) {
    vertx.eventBus().publish("test-feed", userList);
});

File: src/main/java/com/github/nobsust/rltb/TestFeed.java
package com.github.nobsust.rltb;

import io.vertx.core.AbstractVerticle;

/**
 * Created by Sam on 2/8/2017.
 */
public class TestFeed extends AbstractVerticle {
    @Override
    public void start() throws Exception {
        vertx.setPeriodic(2000, h -> vertx.eventBus().publish("test-feed", "from server...."));
    }
}
